src/index.html

```diff
@@ -11,8 +11,62 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__wrapper">
+        <div class="product-card__header">
+          <img
+            src="images/imac.jpeg"
+            alt="Apple laptop"
+            class="product-card__img"
+          />
+        </div>
+        <div class="product-card__main product-info">
+          <div class="product-info__titles">
+            <h3 class="product-info__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h3>
+            <h5 class="product-info__subtitle">Product code: 195434</h5>
+          </div>
+          <div class="reviews">
+            <div class="reviews__stars">
+              <div class="reviews__star"></div>
+              <div class="reviews__star"></div>
+              <div class="reviews__star"></div>
+              <div class="reviews__star"></div>
+              <div class="reviews__star reviews__star--inactive"></div>
+            </div>
```

Comment:
- Rewrite stars block from the [Stars task](https://github.com/mate-academy/layout_stars) with SCSS and use it
[x ] all stars--N modifiers work as expected (Highlight first N stars)

Reply:
ok